### PR TITLE
docs: update cdk overrides doc with delete resource example.

### PR DESCRIPTION
**Pull request by KollaAdithya**
Address #5149

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by efekarakus — APPROVED** [review]

File: site/content/docs/developing/overrides/cdk.en.md [hunk 1]
```diff
@@ -173,4 +174,35 @@ The following example showcases how you can add a property for only a particular
             }
         }
     }
-    ```
\ No newline at end of file
+    ```
+
+The following example showcases how you can delete the default log group created by copilot that holds service logs.
```
Comment (huanjani) [inline site/content/docs/developing/overrides/cdk.en.md]:
```suggestion
The following example showcases how you can delete the default log group, created by Copilot, that holds service logs.
```

File: site/content/docs/developing/overrides/cdk.en.md [hunk 2]
```diff
@@ -80,6 +80,7 @@ In this example, you can view how to:
 - Delete a resource property.
 - Create new resources.
 - Modify a property of an existing resource.
+- Delete a resource.
```
Comment (huanjani) [inline site/content/docs/developing/overrides/cdk.en.md]: But this action isn't part of this first example, right? 🤔

Comment (huanjani) [reply site/content/docs/developing/overrides/cdk.en.md]: Maybe delete this bullet point here, then change the below to 
"The following example showcases how you can delete a resource-- here, the Copilot-created default log group that holds service logs." 
or something like that?

**Review by huanjani — APPROVED** [review]
Thanks for adding this example!

**Pull request merged**